introduction.md

```diff
@@ -1 +1,8 @@
-Hi, my name is Michael and my favortite CodeSchool path so far is Ruby.
\ No newline at end of file
+Hi, my name is Michael and my favortite CodeSchool path so far is Ruby.
+
+Deadly Skills
+* Git
+* Ruby
+* Rails
+* Python
+* Javascript
\ No newline at end of file
```

Comment:
Since you're becoming a GitHub master, could you also add "Killing history using git rebase" as one of your deadly skills?